### ISSUE TITLE
not initialized warning for some weights

### ISSUE THREAD

**Issue by DiddyC** [NormXU/ERNIE-Layout-Pytorch#16]
Hi,
I tried following your readme but I'm getting this error:
Some weights of ErnieLayoutForTokenClassification were not initialized from the model checkpoint at /opt/omniai/work/instance1/jupyter/hugging_face_models/ernie-layout/torch-model/lockbox_torch/pytorch_model.bin and are newly initialized: ['ernie_layout.visual.backbone.batch_norm1.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.1.batch_norm2.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.2.batch_norm3.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.0.batch_norm1.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.2.batch_norm2.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.1.batch_norm3.num_batches_tracked', 'classifier.bias', 'ernie_layout.visual.backbone.resnet.layer0.1.batch_norm1.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.0.batch_norm3.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.0.batch_norm2.num_batches_tracked', 'classifier.weight', 'ernie_layout.visual.backbone.resnet.layer0.0.shortcut.1.num_batches_tracked', 'ernie_layout.visual.backbone.resnet.layer0.2.batch_norm1.num_batches_tracked']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.

I used your weights to load the model and also tried it on my finetuned model. 
Any idea how to get around this?
Thanks!

**Comment by logan-markewich** [comment]
@DiddyC I might be wrong, but I think the visual back-bone is supposed to be newly initialized? I remeber LayoutLMV2/3 gave a similar warning

Since you fine-tuned it, it should be fine too

**Comment by DiddyC** [comment]
@logan-markewich thanks! 
it doesn't sound right to me... Especially if I've already trained it using the paddle logic, I'd want the model to be ready to go after conversion to Torch. Perhaps it's not possible, but if so, what's the point in using a pre trained model?

**Comment by logan-markewich** [comment]
@DiddyC yea i guess im not sure. well, at least the transformer layers are trained 😅 that's the intensive part

**Comment by NormXU** [comment]
@DiddyC I was trying to find weights with the name `batch_norm2.num_batches_tracked` but it seems that these weights are missing in the paddle model as well.

PyTorch doesn't have `num_batches_tracked` in its previous version until a certain updates.

According to the [Pytorch code](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py#L148)  `num_batches_tracked` (starting from zero) will plus 1 for each mini-batch, which means there are two ways to calculate running mean and variance, by exponential method or by `num_batches_tracked`. Paddle doesn't have such a num_batches_tracked and only supports exponential according to its [doc] (https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/layer/norm.py#L799)

So I think if you are using the model for fine-tuning tasks, that's fine.

**Comment by DiddyC** [comment]
@NormXU  Thanks for checking, and for the entire work!
I had another question: won't it be better to use Paddle's tokenizer logic for fine-tuning? since all the benefit is in the pre trained model's weights, what's the point in starting from scratch with different a different token system besides using Ernie-layout's architecture?

**Comment by DiddyC** [comment]
@NormXU thanks, it worked well.
